pages/api/cosign.ts

```diff
@@ -33,13 +36,22 @@ export default async function handler(
       updatedCosigns = [...cosigns, cosignerWallet];
     } else updatedCosigns = [cosignerWallet];
 
+    // if we now have 5, mint this on Zora
+    if (updatedCosigns.length === 5) {
+      console.log("Sending over submission to Minting Cloud Function");
+      const res = await fetch(
+        `${process.env.MORALIS_SERVER_URL}/functions/${cloudFunctionName}?_ApplicationId=${process.env.MORALIS_APP_ID}`
```

Comment:
Shouldn't this be a function?

Reply:
Otherwise looks pretty good, easy to understand code. Couldn't spot anything else.